.appveyor.yml

```diff
@@ -24,7 +24,7 @@ environment:
 #      BUILDDIR: build       # HPC-GAP only supports kernel extensions for of out-of-tree builds
 
 cache:
-  - bootstrap-pkg-full.tar.gz
```

Comment:
The commit "Use package archives for the master branch for bootstrapping" really should be first in this PR.

Also, what exactly is the difference between those two tarballs? Is `bootstrap-pkg-full.tar.gz` th packages from "the last stable release", and `packages-master.tar.gz` "the latest versions of packages we have", or what?  If so, shouldn't `bootstrap-pkg-full.tar.gz` better be called `packages-stable.tar.gz`?

Reply:
@fingolfin do you want to squash commits or rebase interactively and reorder them? Commits are atomic and should work in the order you suggest too.

`bootstrap-pkg-full.tar.gz` is a symlink pointing to `packages-v4.8.8.tar.gz`, so it contains packages from "the last stable release". We also have `packages-stable-4.8.tar.gz` which is at this moment same as `packages-v4.8.8.tar.gz`.

It may be sensible idea to point `bootstrap-pkg-full.tar.gz` to `packages-stable-4.8.tar.gz` since we may be able to put there also some package updates which are not yet in the the last stable release.

Reply:
In addition, `packages-master.tar.gz` has some more packages that require GAP 4.9 and not redistributed with GAP 4.8 - at the moment, besides PrimGrp those are crypting, uuid and ZeroMQinterface.

Reply:
I meant an "interactive rebase": The point is that logically, the commits before the "Use package archives [...]" depend on it (w/o it, their Travis tests would fail).